tests/test_converter.py

```diff
@@ -327,6 +333,16 @@ def test_one_qubit_parametrized_operations_supported_by_pennylane(self, recorder
         assert recorder.queue[3].params == [angle]
         assert recorder.queue[3].wires == single_wire
 
+        assert recorder.queue[4].name == 'U2'
+        assert len(recorder.queue[4].params) == 2
+        assert recorder.queue[4].params == [0.3, 0.4]
+        assert recorder.queue[4].wires == [0]
+
+        assert recorder.queue[5].name == 'U3'
+        assert len(recorder.queue[5].params) == 3
+        assert recorder.queue[5].params == [0.3, 0.4, 0.2]
+        assert recorder.queue[5].wires == [0]
+
```

Comment:
These were previously in the ``test_operations_transformed_into_qubit_unitary``, now that PL core has them, they could be reorganized